CHANGELOG.md

```diff
@@ -21,9 +21,7 @@
   Follow the documentation for the examples and migration details.
 * The `group.id`, `config.storage.topic`, `offset.storage.topic`, and `status.storage.topic` fields in `.spec.config` section of the `KafkaConnect` resource are deprecated and will be forbidden in the `v1` CRD API.
   Please use the new `.spec.groupId`, `.spec.configStorageTopic`, `.spec.offsetStorageTopic`, and `.spec.statusStorageTopic` fields instead.
-* The `.spec.connectCluster`, `.spec.clusters`, `.spec.mirrors[].sourceCluster`, `.spec.mirrors[].targetCluster`, and `.spec.mirrors[].heartbeatConnector` fields of the `KafkaMirrorMaker2` resource are deprecated and will be removed in the `v1` CRD API.
-  Please use the new fields `.spec.target` and `.spec.mirrors[].source` instead to configure the source and target clusters.
-  If you want to deploy and run the Heartbeat connector, you can use separate `KafkaConnect` and `KafkaConnector` custom resources.
```

Comment:
This should be kept there - I guess it's just a wrong rebase?

Reply:
oh no, I did via the git UI (I thought I accepted all the current changes and just re-appended my line to it) , it seemed to have messed up! Thanks for spotting it. I will fix it.